From 47c3c2bf47429021fb9238b9e72b91df6028bca3 Mon Sep 17 00:00:00 2001
From: Abhishek Kumar <abhishek4.kumar@intel.com>
Date: Mon, 4 May 2020 11:59:39 +0530
Subject: [PATCH] egl : Limit egl to 1.4

Android P supports only egl 1.4,
so limit egl 1.4 ver for android api level <=28.

Change-Id: I719a59adeab23ea9c72d270e43b017f190b86fc9
---
 src/egl/main/eglapi.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/egl/main/eglapi.c b/src/egl/main/eglapi.c
index 51557be0864a..add4215e78e4 100644
--- a/src/egl/main/eglapi.c
+++ b/src/egl/main/eglapi.c
@@ -597,6 +597,10 @@ _eglComputeVersion(_EGLDisplay *disp)
        disp->Extensions.KHR_gl_colorspace &&
        disp->Extensions.KHR_surfaceless_context)
       disp->Version = 15;
+
+#if defined(ANDROID_API_LEVEL) && ANDROID_API_LEVEL <= 28
+   disp->Version = 14;
+#endif
 }
 
 /**
-- 
2.24.1

